src/elements/Progressbar/style.scss

```diff
@@ -0,0 +1,22 @@
+:local {
+  @import '../../styles/config';
+  $progress-bar-height: 13px;
+  $bar-background-color: #0290d7;
```

Comment:
please use standardized color from config.scss.

Reply:
I set this colors by myself, but might ask to Jackie for standard color codes later.